conda-recipe/pre-unlink.bat

```diff
@@ -1,4 +1,4 @@
 @echo off
-(
-  "%PREFIX%\python.exe" -m nb_conda_kernels.install --disable
-) >>"%PREFIX%\.messages.txt" 2>&1
+:: Set PATH explicitly as it may not be set correctly by some versions of conda
```

Comment:
I need to understand this a little better, @jjhelmus. Your comment seems to make it clear that the problem is with conda, not with this package. Why not just assume a correctly functioning conda?

One reason I am concerned about this is that I have found that conda's path manipulation scripts don't necessarily do a great job of handling things right when there is a combination of manual path manipulation _and_ conda's own activation mechanism. For instance, conda build effectively "double-activates" the test environment, and that throws conda off if you're trying to do activations as part of your tests (which will be happening here).

Reply:
The comment on "some versions of conda" is forward thinking.  Currently ALL versions of conda do not set PATH when calling post link scripts (see conda/conda#7789).  The reason that this did not cause issues earlier is that on Windows the Python interpreter was patched to add "%PREFIX%\Library\bin" to path.  This patch was dropped from Anaconda's builds of Python 3.7.  The net result is that the post install script will fail most cases when `nb_conda_kernels` is included in a Python 3.7 environment.  More details can be found in [this comment](https://github.com/AnacondaRecipes/nb_conda_kernels-feedstock/issues/1#issuecomment-459873652)

Future versions should fix this bug, see conda/conda#8229

Reply:
OK, I understand now and support this change.